packages/react/src/alert-dialog/AlertDialog.css.ts

```diff
@@ -0,0 +1,78 @@
+import { theme } from "../styles";
+import { type RecipeVariants, recipe, style } from "../vanilla-extract";
+
+export const content = recipe({
+  base: [
+    {
+      bg: "white",
+      rounded: "lg",
```

Comment:
let's add maxW=full here
```suggestion
      maxW: "full",
      rounded: "lg",
```